workflows/pipe-common/scripts/mount_storage.py

```diff
@@ -277,6 +272,17 @@ def build_mount_command(self, params):
                '-o allow_other ' \
                '{mount_options}'.format(**params)
 
+    def __resolve_azure_blob_service_url(self):
+        # add resolved ip address for azure blob service to /etc/hosts (only once per account_name)
+        account_name, _, _ = self._get_credentials(self.storage)
+        command = 'etc_hosts_clear="$(sed -E \'/.*{account_name}.blob.core.windows.net.*/d\' /etc/hosts)" ' \
+                  '&& cat > /etc/hosts <<< "$etc_hosts_clear" ' \
+                  '&& getent hosts {account_name}.blob.core.windows.net >> /etc/hosts'.format(account_name=account_name)
+        exit_code, _, stderr = common.execute_cmd_command_and_get_stdout_stderr(command, silent=True)
+        if exit_code != 0:
```

Comment:
Is it the only place when shall handle shell errors?
E.g. we've experienced an issue with blobfuse installation for Debian (in the background it threw 404) and it was hard to troubleshoot without any log message

Reply:
In this script we already have checks for other places where shell is executed:
 - execute_mount method
 - create_directory method
 - execute_and_check_command method

Reply:
In this case, what log will I get if blobfuse installation fails for whatever reason? E.g. when the following is executed:
https://github.com/epam/cloud-pipeline/blob/2dd2eae60f3fb8779a74e7751014fc64a79b79ae/workflows/pipe-common/scripts/mount_storage.py#L237

Reply:
lines from stdout and stderr will be printed by python process to the current shell, this won't be appear in a pipeline logs. Do you refer me to #151 and propose to resolve this issue in the current PR?

Reply:
Yes

Reply:
ok, done, now stderr of all installation script (install_s3_fuse_goofys, install_azure_fuse_blobfuse etc) will be passed to pipeline as warn logs